src/server/ocpp/services/OCPPService.ts

```diff
@@ -277,7 +279,12 @@ export default class OCPPService {
         error.params.source = headers.chargeBoxIdentity;
       }
       await Logging.logActionExceptionMessage(headers.tenantID, ServerAction.BOOT_NOTIFICATION, error);
-      // FIXME: charging stations in DB with rejected boot notification are still seen as online and usable. OCPP spec is clear on how to handle such case: close websocket, no messages of any kind sent or received, ...
+      if (chargingStation) {
+        // Set boot notification registration status to 'Rejected'
+        chargingStation.registrationStatus = RegistrationStatus.REJECTED;
+        // Save Charging Station
+        await ChargingStationStorage.saveChargingStation(headers.tenantID, chargingStation);
```

Comment:
I don't think you can save a Charging Station with the same ID that was rejected because the s/n was different.
You will make a valid charging station Rejected.
So the ID must differs to save it and you will have no trace of it.
If this is the only goal of this PR then it can close it ;-).

Reply:
Saving just the status instead of the whole data structure will be enough.

Reply:
This does not change the initial problem: 2 charging stations with the same ID with different S/N -> The wrong one must be rejected and you should not override the registration status of the right one.

Reply:
Read the code in the PR.